fea-rs/src/compile/compile_ctx.rs

```diff
@@ -641,6 +641,14 @@ impl<'a, F: FeatureProvider, V: VariationInfo> CompilationCtx<'a, F, V> {
                 self.error(replace.unwrap().range(), "cannot sub glyph by glyph class");
                 None
             }
+            // treat singleton class as single glyph, per the spec:
+            // "If the replacement is a singleton glyph class, then the rule
+            // will be treated identically to a format B rule":
+            // http://adobe-type-tools.github.io/afdko/OpenTypeFeatureFileSpecification.html#5a-gsub-lookuptype-1-single-substitution
```

Comment:
Optional, it would be nice to write an example as some readers [self] don't know what a format B rule is off the top of their head

Reply:
I worry if we follow this practice in general we will end up completely reproducing the spec in our docs; if someone is curious the full section from which this is extracted is available at the provided link.